### PR TITLE
Javascript is added

### PR DESCRIPTION
**Pull request by srujith-reddy**
I think there is no problem with the javascript you wrote. It's just insufficient.
I have added the code that is required for the proper functioning of the app . 
you can modify the design now.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,19 +1,72 @@
...
+    let x=type_convert(firstNum.value);
+    let y=type_convert(secondNum.value);
+    let ans=x-y;
+    console.log(ans);
+    result_window.innerHTML=ans;
+
+})
+
+
+mulitply.addEventListener('click',function(){
```
Comment (Boby900) [inline index.js:31]: thanks @srujith-reddy , but mistakenly you wrote the incorrected spelling of multipy, which is making the  code buggy

Comment (Boby900) [inline index.js]: thanks @srujith-reddy , but mistakenly you wrote the incorrected spelling of multiply, which is making the code buggy

**Review by Boby900 — CHANGES_REQUESTED** [review]

**Comment by srujith-reddy** [comment]
Oh, I am sorry , I didn't test that thing , I tested addition and
subtraction , I will change multiplication as well.

**Review by Boby900 — APPROVED** [review]
thanks @srujith-reddy  for making my calculator up and running

**Pull request merged**